src/instant.rs

```diff
@@ -36,12 +36,6 @@ impl Instant {
     /// Returns the amount of time elapsed from another instant to this one,
     /// or zero duration if that instant is later than this one.
     ///
-    /// # Panics
-    ///
-    /// Previously we panicked if `earlier` was later than `self`. Currently, this method saturates
-    /// to follow the behavior of the standard library. Future versions may reintroduce the panic
-    /// in some circumstances.
-    ///
```

Comment:
This seems copied from std's doc. IIUC fastant provide monotonically non-decreasing clock, so it should be impossible to happen.